src/HPolyhedron.jl

```diff
@@ -454,6 +454,29 @@ function linear_map(M::AbstractMatrix{N},
     return PT(constraints)
 end
 
+# specific an_element function for rational inputs
+function an_element(P::HPolytope{N};
+                    backend=nothing) where {N<:Rational{Int}}
+    # use the floating point method, then convert to rational and check membership
+    p = invoke(an_element, Tuple{HPolytope}, P)
```

Comment:
I am quite convinced that this would be an infinite loop.

Reply:
Haha i tried it! `Tuple{HPolytope}` uses the `Float64` constructor.

Reply:
Okay, then I do not like that exploit :smile: